modules/test/playwright/tests/frontend-data-set-views-web/0_dataSetsSetup.spec.ts

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
@markocikos @dsanz I've realized that in a brand new environment (CI) nobody navigated to the Data Sets page, so DB tables and endpoints do not exist. The first test (alphabetical order) that tries to call an API endpoint without navigating through the UI will fail.

To solve this problem, and until we get the DB updated when the portal starts, I've created this `0_` based spec so it navigates to the Data Sets page and setup the environment properly. 

It is a bit hacky, so please, share your thoughts and possible strategies.

Reply:
Hmm I'm a bit surprised we didn't run into this issue sooner. I guess the test files are always executed alphabetically, so `creationActions` always did the heavy lifting.

This seems to me like a good case for a new fixture, that we include in all DSM tests. Something like `dataSetManagerTest.ts`. In it, we go to Data Set page and confirm something in UI.

Reply:
We usually navigate from the home page to the Data Set using the Control Panel. Loading the Data Sets page does the job.

Now that we are using the API to populate the DB we are skipping some navigation. I'll give a try to the fixture.

Reply:
Yeah, but it was curious to me that for example `visualizationModes.spec.ts` never failed for that reason, although it's using only API. That means it always executed after other test files. It's probably alphabetical, i.e. `creationActions.spec.ts` running first and doing the landing on Data Sets page. In theory, we could rely on the alphabetical order, and that is what you did with `0_` prefix, but that might bite us in the butt later down the line. Playwright might change the logic, like implement parallelization on test file level, or we could simply rename files. It's safer imo to run it on every test file.

Reply:
The order is alphabetical, see https://playwright.dev/docs/test-parallel#control-test-order. You could try only run `visualizationModes.spec.ts` in a clean DB environment.

Reply:
Yeah, it would fail for sure.

Reply:
Got it working with a fixture!! Right now, it is scoped to each test and needs to run after the Login one. Maybe, we could merge the Login with this one.

Reply:
LGTM! I'd only add it to all DSM test files, not just `creationActions`. The reasoning is from above: we might implement parallelization on test file level, or rename files down the line.

> Maybe, we could merge the Login with this one.

No need I think, login is applicable outside of DSM. Composability is one of the main selling point for fixtures after all :slightly_smiling_face: